src/encoded/static/components/filegallery.js

```diff
@@ -331,12 +330,14 @@ FileTable.contextTypes = {
 FileTable.procTableColumns = {
     visualize: {
         title: 'Visualize',
+        hide: (list, columns, meta) => meta && meta.browserOptions && (meta.browserOptions.browserFileSelectHandler === null || meta.browserOptions.currentBrowser !== 'hic'),
         display: (item, meta) => {
             const selectedFile = meta.browserOptions.selectedBrowserFiles.find(file => file['@id'] === item['@id']);
             const selectable = visFileSelectable(item, meta.browserOptions.selectedBrowserFiles, meta.browserOptions.currentBrowser);
             return (
                 <div className="file-table-visualizer">
-                    <input name={item['@id']} type="checkbox" checked={!!selectedFile} disabled={!selectable && !selectedFile} onClick={meta.browserOptions.browserFileSelectHandler} />
+                    // https://stackoverflow.com/a/39020641/178550
+                    <input name={item['@id']} type="checkbox" checked={!!selectedFile} disabled={!selectable && !selectedFile} onClick={meta.browserOptions.browserFileSelectHandler} onChange={() => {}} />
```

Comment:
Should we instead change the existing onClick to an onChange? That should work I would think, and then we wouldn’t need the empty function.

Reply:
@forresttanaka Updated. You are right; my way could be confusing in the future and should be addressed even if unrelated to the ticket. The change looked fine to me on the UI.